beta-src/src/utils/map/drawConvoyOrders.ts

```diff
@@ -24,15 +24,23 @@ export default function drawConvoyOrders(
             const { id, unitID } = originalOrder;
 
             const unitBeingConvoyed = maps.territoryToUnit[fromTerrID];
-            const unitBeingConvoyedOrder = maps.unitToOrder[unitBeingConvoyed];
+            const unitBeingConvoyedOrder =
+              maps.unitToOrder[unitBeingConvoyed] || orderID;
```

Comment:
Why do we need `|| orderID` here?

Reply:
Because maps.unitToOrder only maps "your units" to an order. There is no order for a foreign unit.

Reply:
Might that be what is causing the issue where you can click on other player's fleets?

Reply:
No the issue was in processUnitClick. If the 2nd click after a fleet click wasn't an Army unit it would run startNewOrder.